### PR TITLE
docs: readme update

### PR DESCRIPTION
**Pull request by drwnz**
## PR Type

- Improvement

## Related Links

[TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT1-2587)

## Description

Updates README to be consistent with current driver implementation.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets

## CI Checks

- **Build and test for PR**: Required to pass before the merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,28 +52,28 @@ You should ideally provide a config file for your specific sensor, but default o
...
-| Hesai        | Pandar XT32M  | PandarXT32M  | PandarXT32M.yaml   | Limited         |
-| Hesai        | Pandar QT64   | PandarQT64   |                    | Limited         |
-| Hesai        | Pandar QT128  | PandarQT128  | PandarQT128.yaml   | Limited         |
-| Hesai        | Pandar AT128  | PandarAT128  | PandarAT128.yaml   | Complete        |
-| Hesai        | Pandar 128E4X | Pandar128E4X |                    | Limited         |
-| Velodyne     | VLP-16        | VLP16        | VLP16.yaml         | Limited         |
-| Velodyne     | VLP-16-HiRes  | VLP16        |                    | Limited         |
-| Velodyne     | VLP-32        | VLP32        | VLP32.yaml         | Limited         |
-| Velodyne     | VLS-128       | VLS128       | VLS128.yaml        | Limited         |
+| Manufacturer | Model         | sensor_model | Configuration file | Test status |
```
Comment (aohsato) [inline README.md:55]: [nits] how about using emoji ✅ for describing test status?

File: README.md [hunk 2]
```diff
@@ -52,28 +52,28 @@ You should ideally provide a config file for your specific sensor, but default o
```
Comment (aohsato) [inline README.md:52]: [must] please add your cool description for nebula driver on top-level of this readme.

File: README.md [hunk 3]
```diff
@@ -182,4 +182,4 @@ Parameters shared by all supported models:
 
 ## Diagrams
```
Comment (aohsato) [inline README.md:183]: [nits] Software design? "Diagrams" is an abstract title.

File: README.md [hunk 4]
```diff
@@ -52,28 +52,28 @@ You should ideally provide a config file for your specific sensor, but default o
...
+| Manufacturer | Model         | sensor_model | Configuration file | Test status |
+| ------------ | ------------- | ------------ | ------------------ | ----------- |
+| Hesai        | Pandar 64     | Pandar64     | Pandar64.yaml      | Complete    |
+| Hesai        | Pandar 40P    | Pandar40P    |                    | Complete    |
+| Hesai        | Pandar XT32   | PandarXT32   | PandarXT32.yaml    | Complete    |
+| Hesai        | Pandar XT32M  | PandarXT32M  | PandarXT32M.yaml   | Limited     |
+| Hesai        | Pandar QT64   | PandarQT64   |                    | Limited     |
+| Hesai        | Pandar QT128  | PandarQT128  | PandarQT128.yaml   | Limited     |
+| Hesai        | Pandar AT128  | PandarAT128  | PandarAT128.yaml   | Complete    |
+| Hesai        | Pandar 128E4X | Pandar128E4X |                    | Limited     |
```
[the remark this answers is not in the document]
Comment (aohsato) [reply README.md:64]: Additionally, please explain why no configuration file in fields of 40P/QT64/128E4X?

**Review by aohsato — COMMENTED** [review]
Great work, I added some review comments.

File: README.md [hunk 5]
```diff
@@ -84,7 +84,7 @@ Parameters shared by all supported models:
 
 ### Hesai specific parameters
```
[the remark this answers is not in the document]
Comment (amc-nu) [reply README.md:85]: Fixed in [6869290f7f49375203bf0d553301f8fe792676eb](https://github.com/tier4/nebula/commit/6869290f7f49375203bf0d553301f8fe792676eb)

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (amc-nu) [reply README.md]: Added missing files in [c3e8e29888c73f494392e47a3db0e9ff8cfa78e4](https://github.com/tier4/nebula/commit/c3e8e29888c73f494392e47a3db0e9ff8cfa78e4)

File: README.md (hunk 2 above)
Comment (amc-nu) [reply README.md:56]: Added in https://github.com/tier4/nebula/commit/7f5c0866d90470ee9e9a8ebcb568ec73002ebc95

File: README.md (hunk 1 above)
Comment (drwnz) [reply README.md:80]: Added in https://github.com/tier4/nebula/pull/28/commits/5e82c3b9d9b91382efdbdaa598bed7bbe72e01d1

**Review by amc-nu — APPROVED** [review]

**Pull request merged**